<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BotomMenuPlayer</class>
 <widget class="QWidget" name="BotomMenuPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>53</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QPushButton" name="play_pause">
     <property name="icon">
      <iconset resource="botomMenuPlayer.qrc">
       <normaloff>:/widget/botomMenu/botomMenuPlayer/img/player/play.png</normaloff>
       <normalon>:/widget/botomMenu/botomMenuPlayer/img/player/pause.png</normalon>:/widget/botomMenu/botomMenuPlayer/img/player/play.png</iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="stepDown">
     <property name="icon">
      <iconset resource="botomMenuPlayer.qrc">
       <normaloff>:/widget/botomMenu/botomMenuPlayer/img/player/stepDown.png</normaloff>:/widget/botomMenu/botomMenuPlayer/img/player/stepDown.png</iconset>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="stepUp">
     <property name="icon">
      <iconset resource="botomMenuPlayer.qrc">
       <normaloff>:/widget/botomMenu/botomMenuPlayer/img/player/stepUp.png</normaloff>:/widget/botomMenu/botomMenuPlayer/img/player/stepUp.png</iconset>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTimeEdit" name="positionTme">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QTimeEdit{
background: transparent;
color:white;
}
</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="displayFormat">
      <string notr="true">H:mm:ss</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Widget::Slider" name="sliderVideo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>175</width>
       <height>0</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="pageStep">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTimeEdit" name="durationTme">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QTimeEdit{
background: transparent;
color:white;
}
</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter|Qt::AlignTop</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="displayFormat">
      <string notr="true">H:mm:ss</string>
     </property>
     <property name="time">
      <time>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="soundOn_Of">
     <property name="icon">
      <iconset resource="botomMenuPlayer.qrc">
       <normaloff>:/widget/botomMenu/botomMenuPlayer/img/player/soundOn.png</normaloff>
       <normalon>:/widget/botomMenu/botomMenuPlayer/img/player/soundOff.png</normalon>:/widget/botomMenu/botomMenuPlayer/img/player/soundOn.png</iconset>
     </property>
     <property name="shortcut">
      <string notr="true"/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Widget::Slider" name="sliderAudio">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>76</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>118</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedControls">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widget::Slider</class>
   <extends>QSlider</extends>
   <header>widget/slider/slider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="botomMenuPlayer.qrc"/>
 </resources>
 <connections/>
</ui>
